### PR TITLE
Nexus playbook

### PR DESCRIPTION
**Pull request by Legamanchik**
Playbook for install nexus but you need to update packages first

### CODE REVIEW

File: nexus.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+- name: Nexus
+  hosts: your_gcp_vm_group
```
Comment (arkma) [inline nexus.yml:3]: You may forgot to specify inventory group. It would better to use `server_nexus` group.

File: nexus.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    httpd_ssl_certificate_file: 'files/nexus.vm.crt'
+    httpd_ssl_certificate_key_file: 'files/nexus.vm.key'
+    # httpd_ssl_certificate_chain_file: "{{ httpd_ssl_certificate_file }}"
+    httpd_copy_ssl_files: false
+    nexus_local_users:
+      - username: jenkins # used as key to update
+        first_name: Jenkins
+        last_name: CI
+        email: support@company.com
+        password: "s3cr3t"
```
Comment (arkma) [inline nexus.yml:34]: Move credentials to environment variable to be able to inject them via AWX

File: nexus.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    nexus_blobstores:
+      - name: docker-images
+        path: /var/nexus/blobs/company-artifacts
+    nexus_repos_docker_hosted:     
+      - name: blue-docker-repo
+        blob_store: docker-images
+
+  roles:
+    - { role: geerlingguy.java, java_packages: openjdk-8-jre-headless}
+    - { role: ansible-thoteam.nexus3-oss, tags: ['ansible-thoteam.nexus3-oss'] }
```
Comment (arkma) [inline nexus.yml:46]: I don't see this role in the `ansible-galaxy`. Where can we get this role?

**Review by arkma — CHANGES_REQUESTED** [review]
Where is your inventory file? Do you use GCP dynamic inventory?
How can I verify that this ansible playbook installs nexus correctly?

**Review by arkma — APPROVED** [review]

**Review by sbodia — APPROVED** [review]

**Pull request merged**